ui/lib/core/addon/components/choose-pgp-key-form.hbs

```diff
@@ -0,0 +1,52 @@
+{{#if this.selectedPgp}}
+  <form id="confirm-pgp-key" {{on "submit" this.handleSubmit}} aria-label="begin token generation with PGP key">
+    <div>
+      <p class="has-bottom-margin-m">
+        Below is the base-64 encoded PGP Key that will be used to encrypt the generated operation token. Next we'll enter
+        portions of the root key to generate an operation token. Click the "Generate operation token" button to proceed.
+      </p>
+      <h4 class="field-title has-bottom-padding-m is-fullwidth">
+        PGP Key
+        {{this.pgpKeyFile.filename}}
+      </h4>
+      <div class="message is-list has-copy-button" tabindex="-1">
+        <HoverCopyButton @copyValue={{this.pgpKey}} />
+        <code class="is-word-break">{{this.pgpKey}}</code>
+      </div>
+    </div>
+    <div class="field is-grouped">
+      <div class="control">
+        <button type="button" class="button" {{on "click" (fn (mut this.selectedPgp) "")}}>
+          Back
+        </button>
+      </div>
+      <div class="control">
+        <button type="submit" disabled={{and (not this.pgpKey)}} class="button is-primary">
```

Comment:
it looks like this disabled conditional was the same in the original hbs file, too - but I think we can remove the `and` here?